api/src/main/java/org/openmrs/module/commonreports/reports/DRCHIVStageReportManager.java

```diff
@@ -169,7 +169,7 @@ public ReportDefinition constructReportDefinition() {
 		hivStage3And4Obs.setValueList(hivStages);
 		hivStage3And4Obs.setTimeModifier(TimeModifier.LAST);
 		
-		// Reffered out
+		// Not Reffered out
```

Comment:
```suggestion
		// Not Refered out
```

Reply:
Process is a "referral" so someone is "referred out" or, more accurately in this case "transferred out".